### DISCUSSION TITLE
why export the reactQuery hooks

### DISCUSSION THREAD

**Discussion by AaronAbuUsama** [wevm/wagmi#1192] [General]
while writing some hooks for my app i see that wagmi also exports the react query hooks. my question is why? is it an oversight, does wagmi modify the tanstack hooks it exports? do i need to even import react query or can i just use whats exposed my wagmi

**Comment by tmm** [comment]
wagmi treats React Query as an internal implementation detail and has a custom implementation of related hooks that folks can use if they want.
